tests/parametric/test_trace_sampling.py

```diff
@@ -344,3 +346,53 @@ def test_trace_dropped_by_trace_sampling_rule_tags(self, test_agent, test_librar
 
         assert span["metrics"].get(SAMPLING_PRIORITY_KEY) == -1
         assert span["metrics"].get(SAMPLING_RULE_PRIORITY_RATE) == 0.0
+
+
+@scenarios.parametric
+@rfc("https://docs.google.com/document/d/1S9pufnJjrsxH6pRbpigdYFwA5JjSdZ6iLZ-9E7PoAic/")
```

Comment:
Should we change this rfc reference for the final rfc version?
I say this because I read this in the referenced document:

> These changes have now been merged into the [original RFC](https://docs.google.com/document/d/1xtKkHSMmzgz7X6ZAozltOU53qO6oI0D_jhWnbvN_k8s/edit?usp=sharing).  Use that RFC instead.

Reply:
I will keep as is, and when the work on the newer RFC is done, we will update everything altogether